### ISSUE TITLE
material "next pass" not get animation

### ISSUE THREAD

**Issue by marcosbitetti** [godotengine/godot#28037]
V 3.2 dev custom_build 3f76d2c (2019-04-14)

Linux, i7, no gpu.

Material, shader usin next pass propertie not work with animation!

Add a next pass material in material properties.

video:
[https://youtu.be/NjFDIu96pao](https://youtu.be/NjFDIu96pao)

**Comment by StraToN** [comment]
Did you test this project with another computer? Possibly with a discrete GPU?

**Comment by marcosbitetti** [comment]
I was able to test in my daughter pc, Q9300 with GeForce GTX630, Win7
Same error!

**Comment by marcosbitetti** [comment]
I found the issue origin!
comment world_vertex_coords:
**render_mode cull_front, specular_disabled;//, world_vertex_coords;**
but world_vertex_coords work fine on the first shader, the problem is in "next pass"

**Comment by KoBeWi** [comment]
Can anyone still reproduce this bug in Godot 3.2.3 or any later release?

If yes, please ensure that an up-to-date Minimal Reproduction Project (MRP) is included in this report (a MRP is a zipped Godot project with the minimal elements necessary to reliably trigger the bug). You can upload ZIP files in an issue comment with a drag and drop.
Triage procedure:

**Comment by Zylann** [comment]
Not easy to guess the exact reproduction steps, but looking at the video, my only guess was that `next_pass` was used to make an outline around the goblins.
So I made a very simple animated mesh, and assigned an outlined material with next pass in `material_override` (done with a script). It correctly animated both the mesh and its outline.
[SimpleSkinnedCube.zip](https://github.com/godotengine/godot/files/6793360/SimpleSkinnedCube.zip)

Now it is possible that the real reproduction may be slightly different, or that the bug got fixed, hard to know without MRP from original report.
